content/en/getting-started/installation.md

```diff
@@ -1,155 +1,206 @@
+If you cannot use one of our prebuilt binary releases of LocalStack, you can install the LocalStack CLI in a Python environment.<br>
+Please make sure to install the following tools on your machine before moving ahead:
+{{% markdown %}}
+- [`python`](https://docs.python.org/3/using/index.html) (Python 3.7 up to 3.11 is supported)
+- [`pip`](https://pip.pypa.io/en/stable/installation/) (Python package manager)
+
+Afterwards you can install the LocalStack CLI in your Python environment with:
+{{< command >}}
+$ python3 -m pip install --upgrade localstack
+{{< / command >}}
+{{% /markdown %}}
 
 {{< alert title="Note" >}}
 To download a specific version of LocalStack, check out our [release page](https://github.com/localstack/localstack) and download it in the following manner:
-
 {{< command >}} 
 $ python3 -m pip install localstack==<version>
 {{< / command >}}
-
+{{% markdown %}}
 Here `<version>` depicts the particular LocalStack version that you would like to download and use.
+{{% /markdown %}}
 {{< /alert >}}
 
 {{< alert title="Important" color="danger" >}}
 Do not use `sudo` or the `root` user - LocalStack should be installed and started entirely under a local non-root user. 
-If you have problems with permissions in MacOS X Sierra, install with `python3 -m pip install --user localstack`.
 {{< /alert >}}
+{{< /tab >}}
+{{< /tabpane >}}
 
-Afterwards you should be able to use the LocalStack CLI in your terminal:
-
-{{< command >}}
-$ localstack --help
-Usage: localstack [OPTIONS] COMMAND [ARGS]...
-
-  The LocalStack Command Line Interface (CLI)
-
-Options:
-...
-{{< / command >}}
-
-#### Updates
-
-The LocalStack CLI also allows you to easily update the different components of LocalStack. To check the various options available for updating, run:
-
+To verify that the LocalStack CLI was installed correctly, you can check the version in your terminal:
 {{< command >}}
-$ localstack update --help
-
-Usage: localstack update [OPTIONS] COMMAND [ARGS]...
-
-  Update LocalStack components
-
-Options:
-  --help  Show this message and exit.
-
-Commands:
-  all             Update all LocalStack components
-  docker-images   Update container images LocalStack depends on
-  localstack-cli  Update LocalStack CLI tools
+$ localstack --version
+{{< localstack-latest-version >}}
 {{< / command >}}
 
-You can decide to update the CLI itself, the LocalStack Docker images, or all at once:
+You are all set!
+To use all of LocalStack's features we recommend to [get a LocalStack account and set up your api key]({{< ref "api-key" >}}).
+Afterwards, check out our [Quickstart guide]({{< ref "quickstart" >}}) to start your local cloud!
 
-{{< command >}}
-$ # Update all components
-$ localstack update all
-$ # Only update the LocalStack docker images
-$ localstack update docker-images
-$ # Only update the LocalStack CLI
-$ localstack update localstack-cli
-{{< / command >}}
-
-#### Starting LocalStack with the LocalStack CLI
+## Alternatives
```

Comment:
Great overview of alternatives with a short summary for each 👍👍

Reply:
This framing also implicitly clarifies that the CLI is our recommended way ✨  (in line with feedback we received at the AWS Switzerland Builder's Day)